### PR TITLE
rblake2sum: init at 0.3.1 

### PR DESCRIPTION
**Pull request by dpc**
## Description of changes

https://github.com/crev-dev/rblake2sum

Basically - it allows you to calculate a digest of a content of a directory, which is very useful, but strangely not well covered with existing tools.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/rblake2sum/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  cargoHash = "sha256-egwL3z7uB4AcRwPT0uPrenyh4FSxhbZKMdkPhRztMbs=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ Security ];
+
+  meta = with lib; {
+    description = "A recursive blake2 digest (hash) of a file-system path";
+    homepage = "https://github.com/crev-dev/rblake2sum";
+    license = [ licenses.mit licenses.mpl20 licenses.asl20 ];
```
Comment (soupglasses) [inline pkgs/tools/misc/rblake2sum/default.nix:25]: I would recommend to just choose one here, for example `mit`, as how this is currently written means all three licensees are applied at the same time. Which is not what the be the meaning upstream.

**Comment by soupglasses** [comment]
Sorry for not spotting this earlier, but it would likely fit better into `pkgs/tools/security` as this is where `b2sum` and `b3sum` is located.

Other than that, this looks good to me. :smile: :+1:

**Comment by soupglasses** [comment]
Result of `nixpkgs-review pr 251689` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rblake2sum</li>
  </ul>
</details>

**Comment by dpc** [comment]
> Result of `nixpkgs-review pr 251689` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

TIL. Thanks.

**Comment by h7x4** [comment]
I think it would be nice if you formatted the "add-to-maintainers-list" commit like this:

maintainers: add dpc

While I haven't seen it stated as a requirement anywhere, it seems to be a kind of convention.

**Review by soupglasses — APPROVED** [review]

**Comment by dpc** [comment]
Since it's been a month, are there any acceptable ways to ask somewhere for someone to approve, or should I just wait patiently?

**Comment by soupglasses** [comment]
There are some posts in https://discourse.nixos.org/ which can get eyes on this. I have taken the courtesy to link this PR in one of these :+1:

**Pull request merged**